modules/web-console/web-console-server/pom.xml

```diff
@@ -108,36 +112,40 @@
             <version>${commons.compress.version}</version>
         </dependency>
 
+        <!-- Ignite dependencies -->
         <dependency>
-            <groupId>io.vertx</groupId>
-            <artifactId>vertx-core</artifactId>
-            <version>${vertx.version}</version>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-core</artifactId>
+            <version>${project.version}</version>
         </dependency>
 
         <dependency>
-            <groupId>io.vertx</groupId>
-            <artifactId>vertx-config</artifactId>
-            <version>${vertx.version}</version>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-web-console-common</artifactId>
+            <version>${project.version}</version>
         </dependency>
 
+        <!-- TODO WC-1006 for quick test from IDE -->
         <dependency>
```

Comment:
ignite-web-console-common already has ignite-rest-http module as dependency. Do we really need this?

Reply:
Fixed.